portfolio/src/main/webapp/script.js

```diff
@@ -32,4 +32,17 @@ function guessMyTrip() {
     document.getElementById('display').innerHTML = 
         "Unfortunately you guessed it wrong!, Try Again!";
   }
+}
+function moreFacts(){
+  const facts = ["I have hosted Jeff Weiner in Trinity College",
+    "I took part in my first ever hackathon with JP Morgan in 2019.",
+    "Once JP Morgan flew me business class to London! (amazing experience)",
+    "I spend a lot of time volunteering in college.",
+    "I would love to have a job which allows me to travel.",
+    "I am preparing for a 5 month backpacking trip !"];
+
+    let rand = Math.floor(Math.random() * facts.length);
```

Comment:
It has extra indentation. https://google.github.io/styleguide/jsguide.html#formatting-block-indentation

Reply:
corrected the indentation now